### PR TITLE
Fix Rest API to check multiple toggles when the toggle does not exist and add potentially some form params

### PR DESCRIPTION
**Pull request by alexandrenavarro**

### CODE REVIEW

**Comment by paul58914080** [comment]
Hello @alexandrenavarro ,

Do you have an example where you have tested this api ?

**Comment by alexandrenavarro** [comment]
In my current application, we use an older ff4j (1.7.x with spring-boot 2.1.x) where the code of the ff4j-boot-starter-is is in java. I did the fix directly with featureUIDs as body and formParams as requestParam and just test it manually, it is now used in production since last thursday. 

I did the equivalent in kotlin for ff4j-spring-boot-web-api and tried to do the same ff4j-web-api, we will wait your integration to upgrade to the last version of ff4j-spring-boot-web-api on our side.
In my opinion, I think we should add some integration tests (IT) on all the api of ff4j (not just this one API) with SpringBootTest and MockMvc for instance (what I generally do on my spring-boot projects). 
Nevertheless we may share these IT between ff4j-spring-boot-web-api and ff4j-web-api to be sure behavior are the same (maybe not SpringBootTest because ff4j-web-api do not use spring-boot).

@clun and @paul58914080 tell me what you want to do IT (I was surprised there is no IT, I set in a previous comment) but the point it is not linked to my add of REST API but more globally how you want to do your IT in ff4j-*-web-api.

@clun we really need to pass featureUIDs and in formParams in our case (we have to pass user as parameter in our case, we previously use the /check/$featuresIds with requestParams).
I did it featureUIDs as body and formParams as requestParams to be coherent the current REST API with the /check/$featuresIds.
I tested with spring-mvc and it works, but effectively, I did something I think it should work in ff4j-web-api (really long time I didn't use jaxrs). I just created a draft PR for jaxrs (not tested but I really think it works) https://github.com/ff4j/ff4j/pull/459 
Tell me what do you think about it?

To summarize  @clun and @paul58914080 
1) Tell me what you want to add IT in ff4j-*-web-api, I think it is more what you want to do globally, I will do/adapt it to test the API I added.
2) Tell me if your are ok to add as formParams in multipleChecks (check my draft PR https://github.com/ff4j/ff4j/pull/459).
   - If your are not ok by the REST API, tell me how to do it to add formParams for multiple checks.
   - If your are ok, I will do the equivalent for the boot-starter (my previous PR in reality).

Thanks in advance,

**Comment by paul58914080** [comment]
Hello @alexandrenavarro ,

Thank you for the brief. We do have IT maybe there is a scope to improve from readability.

I had to re-work a bit on this PR with https://github.com/ff4j/ff4j-spring-boot-starter-parent/pull/147 and as did @clun with https://github.com/ff4j/ff4j/commit/b8fe410b083449905a39703275cc6cd5a0973430#diff-93d8699d81bfdee98fb1fa14534182fa

> Tell me what you want to add IT in ff4j-*-web-api, I think it is more what you want to do globally, I will do/adapt it to test the API I added

**Reference:** https://github.com/ff4j/ff4j-spring-boot-starter-parent/blob/master/ff4j-spring-boot-web-api/src/test/java/org/ff4j/spring/boot/web/api/resources/ff4j/FF4JStepDef.java

**BDD:** https://github.com/ff4j/ff4j-spring-boot-starter-parent/blob/master/ff4j-spring-boot-web-api/src/test/resources/features/FF4JResource.feature#L151-L161

I agree with you on the statement _Nevertheless we may share these IT between ff4j-spring-boot-web-api and ff4j-web-api to be sure behaviour are the same (maybe not SpringBootTest because ff4j-web-api do not use spring-boot)._

> Tell me if your are ok to add as formParams in multipleChecks (check my draft PR ff4j/ff4j#459)

From the RESTful context the featureUIDs should be passed as a query param and the execution context can remain as form param with MultiMap as mentioned in https://github.com/ff4j/ff4j/issues/449#issuecomment-703093099

Example: `/ff4j/check?featureUIDs=bar,qux`

@clun n @alexandrenavarro what do you think ?

**Comment by alexandrenavarro** [comment]
@clun Thank you, I saw you merged my PR yesterday https://github.com/ff4j/ff4j/pull/459, so I supposed your are ok with this solution.
So I did the equivalent for the boot-starter https://github.com/ff4j/ff4j/pull/459

Thanks again.
